### PR TITLE
Add option to filter benchs by avaName and update ini

### PR DESCRIPTION
**Pull request by awirb**
* rename flagAimec to Aimec in statsExample towards #330 
* update benchmark ini

### CODE REVIEW

**Review by matthiasto — APPROVED** [review]

File: avaframe/ana4Stats/getStatsCfg.ini [hunk 1]
```diff
@@ -9,4 +9,4 @@ scenario = test
 # optional - comparison type, e.g. if different distributions are used
 distType =
 # if True also run aimec
-flagAimec = False
+Aimec = False
```
Comment (fso42) [inline avaframe/ana4Stats/getStatsCfg.ini:12]:
```suggestion
aimec = False
```

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -251,7 +251,7 @@ def com1DFAMain(cfg, avaDir):
             # Count total number of simulations
             countRel = countRel + 1
 
-            if cfgPar.getboolean('flagVarEnt') and (simName + '_entres_dfa') in cuSim:
+            if cfgPar.getboolean('VarEnt') and (simName + '_entres_dfa') in cuSim:
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:254]:
```suggestion
            if cfgPar.getboolean('varEnt') and (simName + '_entres_dfa') in cuSim:
```

File: avaframe/com1DFA/com1DFACfg.ini [hunk 3]
```diff
@@ -21,9 +21,9 @@ addTSteps = True
 
 [PARAMETERVAR]
 # True if parameter variation option shall be included
-flagVarPar = False
+parameterVar = False
 # True if entres simulation shall be used for parameter variation
-flagVarEnt = True
+VarEnt = True
```
Comment (fso42) [inline avaframe/com1DFA/com1DFACfg.ini:26]:
```suggestion
varEnt = True
```

File: avaframe/runStatsExample.py [hunk 4]
```diff
@@ -69,8 +69,8 @@
     reportDir = os.path.join(avaDir, 'Outputs', 'com1DFA', 'reports')
     # write report
     gR.writeReport(reportDir, reportDictList, cfgMain['FLAGS'], plotDict)
-    
-    if cfg.getboolean('flagAimec') == True:
+
+    if cfg.getboolean('Aimec') == True:
```
Comment (fso42) [inline avaframe/runStatsExample.py:73]:
```suggestion
    if cfg.getboolean('aimec') == True:
```

File: benchmarks/avaFlatPlaneVarParTest/avaFlatPlaneVarPar_com1DFACfg.ini [hunk 5]
```diff
@@ -23,9 +18,9 @@ addTSteps = False
 
 [PARAMETERVAR]
 # True if parameter variation of MU option shall be included
-flagVarPar = True
+parameterVar = True
 # True if entres simulation shall be used for parameter variation
-flagVarEnt = True
+VarEnt = True
```
Comment (fso42) [inline benchmarks/avaFlatPlaneVarParTest/avaFlatPlaneVarPar_com1DFACfg.ini:23]:
```suggestion
varEnt = True
```

File: benchmarks/avaHelixChannelVarParTest/avaHelixChannelVarPar_com1DFACfg.ini [hunk 6]
```diff
@@ -23,9 +18,9 @@ addTSteps = False
 
 [PARAMETERVAR]
 # True if parameter variation of MU option shall be included
-flagVarPar = True
+parameterVar = True
 # True if entres simulation shall be used for parameter variation
-flagVarEnt = True
+VarEnt = True
```
Comment (fso42) [inline benchmarks/avaHelixChannelVarParTest/avaHelixChannelVarPar_com1DFACfg.ini:23]:
```suggestion
varEnt = True
```

File: benchmarks/avaParabolaVarParTest/avaParabolaVarPar_com1DFACfg.ini [hunk 7]
```diff
@@ -23,9 +18,9 @@ addTSteps = False
 
 [PARAMETERVAR]
 # True if parameter variation of MU option shall be included
-flagVarPar = True
+parameterVar = True
 # True if entres simulation shall be used for parameter variation
-flagVarEnt = True
+VarEnt = True
```
Comment (fso42) [inline benchmarks/avaParabolaVarParTest/avaParabolaVarPar_com1DFACfg.ini:23]:
```suggestion
varEnt = True
```

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/ana4Stats/getStatsCfg.ini (hunk 1 above)
Comment (awirb) [reply avaframe/ana4Stats/getStatsCfg.ini:12]: Done

File: avaframe/com1DFA/com1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:254]: Done

File: avaframe/com1DFA/com1DFACfg.ini (hunk 3 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:26]: Done

File: avaframe/runStatsExample.py (hunk 4 above)
Comment (awirb) [reply avaframe/runStatsExample.py:73]: Done

File: benchmarks/avaFlatPlaneVarParTest/avaFlatPlaneVarPar_com1DFACfg.ini (hunk 5 above)
Comment (awirb) [reply benchmarks/avaFlatPlaneVarParTest/avaFlatPlaneVarPar_com1DFACfg.ini:23]: Done

File: benchmarks/avaHelixChannelVarParTest/avaHelixChannelVarPar_com1DFACfg.ini (hunk 6 above)
Comment (awirb) [reply benchmarks/avaHelixChannelVarParTest/avaHelixChannelVarPar_com1DFACfg.ini:23]: done

File: benchmarks/avaParabolaVarParTest/avaParabolaVarPar_com1DFACfg.ini (hunk 7 above)
Comment (awirb) [reply benchmarks/avaParabolaVarParTest/avaParabolaVarPar_com1DFACfg.ini:23]: done

**Review by fso42 — APPROVED** [review]

**Pull request merged**